### PR TITLE
changed orpheushacks. CNAME again

### PR DESCRIPTION
**Pull request by malteianlauterbach**

### CODE REVIEW

**Comment by lachlanjc** [comment]
This isn’t how DNS works for Vercel :) they have a generic cname.vercel-dns thing you’ll need to use instead

File: hackclub.com.yaml [hunk 1]
```diff
@@ -617,7 +617,7 @@ numberwang:
 orpheushacks: # Harrison Katz automatically wins for adding the DNS entry - thats how it works, right?
 - ttl: 1
   type: CNAME
-  value: 4835f8d6-2c4c-4441-8571-3889c2024016.repl.co.
+  value: orpheushacks2.vercel.app.
```
Comment (lachlanjc) [inline hackclub.com.yaml:620]:
```suggestion
  value: cname.vercel-dns.com.
```

**Comment by lachlanjc** [comment]
You also won’t be able to use Vercel DNS unless the project is owned by HQ’s organization, due to Vercel domain restrictions. I’ve just deployed it on our org & done the DNS setup so it’ll work, & it will autodeploy when you push. Lmk if you need any configuration changed.

**Comment by malteianlauterbach** [comment]
#response_container_BBPPID{font-family: initial; font-size:initial; color: initial;}  Think you so much!                                                                                                                                                                                                                                                                                                                        Von: notifications@github.comGesendet: 2. November 2020 18:34An: dns@noreply.github.comAntworten: reply@reply.github.comCc: malte@enigma-blog.de; author@noreply.github.comBetreff: Re: [hackclub/dns] changed orpheushacks. CNAME again (#408)  

You also won’t be able to use Vercel DNS unless the project is owned by HQ’s organization, due to Vercel domain restrictions. I’ve just deployed it on our org & done the DNS setup so it’ll work, & it will autodeploy when you push. Lmk if you need any configuration changed.



—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Pull request merged**